tests/test_sns.py

```diff
@@ -1,7 +1,7 @@
 import json
-import pytest
-import botocore
 
+import botocore
```

Comment:
I think I understand what you're doing here, separating built-in modules from third party modules, I've been adding a comment before each section to make this clear in my code.  Could you please do the same?